src/tests/ftest/util/dmg_utils.py

```diff
@@ -252,16 +252,18 @@ def storage_format(self, force=False, timeout=30, verbose=False):
         self.timeout = saved_timeout
         return self.result
 
-    def storage_set_faulty(self, uuid, force=True):
+    def storage_set_faulty(self, host, uuid, force=True):
```

Comment:
Also need to update this usage
https://github.com/daos-stack/daos/blob/f9b3f46a3d0e2b6c7555d9e4897d0eba398feca7/src/tests/ftest/control/dmg_storage_query.py#L255

Reply:
good catch, thanks. fixed